core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -138,13 +134,10 @@ describe('Block reverting test', function () {
         // Create test wallets
         tester = await Tester.init(ethClientWeb3Url, apiWeb3JsonRpcHttpUrl, baseTokenAddress);
         alice = tester.emptyWallet();
-        logs = fs.createWriteStream('revert.log', { flags: 'a' });
+        logs = fs.createWriteStream(`revert_${fileConfig.chain}.log`, { flags: 'a' });
     });
 
     step('run server and execute some transactions', async () => {
-        // Make sure server isn't running.
-        await killServerAndWaitForShutdown(tester, serverProcess).catch(ignoreError);
-
```

Comment:
don't forget to kill server here and let's make a test param which will be set to !fileConfig.loadFromFile || !process.env.NO_KILL

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/aff37b4a4842927952987db81d5c2b4bb2ed78f2